scripts/graph_accelerometer.py

```diff
@@ -52,20 +169,42 @@ def setup_matplotlib(output_to_file):
 
 def main():
     # Parse command-line arguments
-    usage = "%prog [options] <log>"
+    usage = "%prog [options] <logs>"
     opts = optparse.OptionParser(usage)
     opts.add_option("-o", "--output", type="string", dest="output",
                     default=None, help="filename of output graph")
+    opts.add_option("-f", "--max_freq", type="float", default=200.,
+                    help="maximum frequency to graph")
+    opts.add_option("-r", "--raw", action="store_true",
+                    help="graph raw accelerometer data")
+    opts.add_option("-c", "--compare", action="store_true",
+                    help="graph comparison of power spectral density "
+                         "between different accelerometer data files")
+    opts.add_option("-s", "--specgram", action="store_true",
+                    help="graph spectogram of accelerometer data")
+    opts.add_option("-a", type="string", dest="axis", default="all",
+                    help="axis to graph (one of 'all', 'x', 'y', or 'z')")
     options, args = opts.parse_args()
-    if len(args) != 1:
+    if len(args) < 1:
         opts.error("Incorrect number of arguments")
 
     # Parse data
-    data = parse_log(args[0])
+    datas = [parse_log(fn) for fn in args]
+
+    if options.output and os.path.splitext(options.output)[1].lower() == '.csv':
+        write_frequency_response(datas, options.output)
+        return
```

Comment:
I think it would be better to add a separate option (eg, `--csv myfile.csv`) than to overload the meaning of the `-o` option.

Reply:
Well, right now the user may specify different extensions already: `-o resonances_x.png` or `-o resonances_x.svg` or `-o resonances_x.pdf`, etc., and the output will be different according to the extension. Of course, you cannot really 'write' a picture into CSV file, but I thought that, from the user perspective, it would feel natural to similarly change the extension and get essentially the same thing in a different format. However, to look less hacky and more natural, I changed the code to also support spectrogram, and added explicit error messages if `--raw` or `--compare` was requested.